apps/desktop/src-electron/libs/shortcuts.ts

```
@@ -7,7 +7,7 @@ import { shortcutsKeys } from '@onekeyhq/shared/src/shortcuts/shortcutsKeys.enum
 export function registerShortcuts(callback: (event: EShortcutEvents) => void) {
   void app.whenReady().then(() => {
     Object.entries(shortcutsMap).forEach(([event, { keys }]) => {
-      if (keys) {
+      if (keys && keys.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)